Firmware/RTK_Surveyor/RTK_Surveyor.ino

```diff
@@ -594,41 +622,70 @@ void setup()
 {
   Serial.begin(115200); //UART0 for programming and debugging
 
+  identifyBoard(); //Determine what hardware platform we are running on
+  log_d("identifyBoard complete");
+
+  initializePowerPins(); //Initialize any essential power pins - e.g. enable power for the Display
+  log_d("initializePowerPins complete");
+
   beginI2C();
+  log_d("beginI2C complete");
 
-  beginDisplay(); //Start display first to be able to display any errors
+  beginDisplay(); //Start display to be able to display any errors
+  log_d("beginDisplay complete");
 
   beginGNSS(); //Connect to GNSS to get module type
+  log_d("beginGNSS complete");
 
   beginFS(); //Start file system for settings
+  log_d("beginFS complete");
 
-  beginBoard(); //Determine what hardware platform we are running on and check on button
+  beginBoard(); //Now finish settup up the board and check the on button
+  log_d("beginBoard complete");
 
   displaySplash(); //Display the RTK product name and firmware version
+  log_d("displaySplash complete");
 
   beginLEDs(); //LED and PWM setup
+  log_d("beginLEDs complete");
```

Comment:
I don't know if I like these. Fine for debugging, but once we get RCs into the hands of users, these extra prints can cause too much noise during boot. I'd rather implement our own debug print with level handler. Lee mentioned we often need the ability for users to turn on debug messages. log_x requires a complete recompile. I think we may need to move to a custom debug_1(), debug_2(), etc (or you pick the function name) that allows us to programmatically control what is printed based on a settings.debugLevel. We will still likely need to set the debug level to log during RCs (so that we can see the debug statements of built-in libraries and such) but our own debug statements should be in our control.

Reply:
Let's do this on a new branch. This is big enough :)

Reply:
I was in the weeds, trying to figure out exactly where the code was silent-crashing. I'm happy to back these extra log_d's out now

Reply:
I have been there *many* times as well, so it's useful, but only in extreme situations.